data/lists/guardianbots.xyz.json

```
@@ -0,0 +1,35 @@
+{
+    "$schema": "../../schema/list.json",
+    "id": "guardianbots.xyz",
+    "added": 1696799561,
+    "name": "Guardian Bot List",
+    "url": "https://guardianbots.xyz/",
+    "icon": "https://cdn.guardianbots.xyz/uploads/logo.png",
+    "language": "English",
+    "display": 1,
+    "defunct": 0,
+    "discord_only": 0,
+    "description": "Discover hundreds of Discord bots and add your own to make it popular",
+    "api_docs": null,
+    "api_post": null,
+    "api_post_method": null,
+    "api_field": null,
+    "api_shard_id": null,
+    "api_shard_count": null,
+    "api_shards": null,
+    "api_get": null,
+    "api_all": null,
+    "view_bot": "https://guardianbots.xyz/bots/:id",
+    "bot_widget": null,
+    "content": "Bot list is opperational.",
+    "owners": "@whatisyourname. (878850717706756116)",
```

```suggestion
    "owners": "@whatisyourname.#0000 (878850717706756116)",
```
Looks like the validation isn't set up for new usernames yet, so we can just do what Discord does and use a #0000 discrim.